public/app/features/provisioning/Wizard/BootstrapStep.test.tsx

```diff
@@ -198,7 +198,7 @@ describe('BootstrapStep', () => {
 
       setup();
 
-      expect(await screen.findByText('7 resources')).toBeInTheDocument();
+      expect((await screen.findAllByText('7 resources')).length).toBeGreaterThan(0);
```

Comment:
```suggestion
      expect(await screen.findAllByText('7 resources')).toHaveLength(2);
```
Also a comment explaining where there are now 2 resource strings on the page would be helpful.